README.md

```diff
@@ -22,3 +22,10 @@ A simple API that powers the catalog
 ```bash
 npm run typeorm-run
 ```
+
+## Elasticsearch
```

Comment:
This will be used for full text search across the entire database at some point.